README.md

```diff
@@ -1,185 +1,217 @@
 # Planning Inspectorate Back Office
 
-> This is the Planning Inspectorate Back Office monorepo that holds the API and WEB apps plus any additional required packages.
+This is the Planning Inspectorate Back Office monorepo that contains all the apps for running the back office. 
 
-- [Planning Inspectorate Back Office](#planning-inspectorate-back-office)
-	- [Installing / Getting started](#installing--getting-started)
-	- [Developing](#developing)
-		- [Built With](#built-with)
-		- [Prerequisites](#prerequisites)
-		- [Setting up Dev](#setting-up-dev)
-		- [Setting up Database locally](#setting-up-database-locally)
-		- [Building](#building)
-		- [Deploying / Publishing](#deploying--publishing)
-		- [Docker](#docker)
-			- [API](#api)
-			- [Web](#web)
-	- [Configuration](#configuration)
-	- [Tests](#tests)
-	- [Style guide](#style-guide)
-	- [Licensing](#licensing)
+The back office system includes a JSON API, which retrieves data from a database, and a web front-end (utilising [server-side rendering](https://web.dev/rendering-on-the-web/#server-rendering)). There are also some [Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview) for background tasks (such as virus scans), [Azure Blob Storage](https://azure.microsoft.com/en-gb/products/storage/blobs) is used for documents, and [Azure Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) for integration.
 
-## Installing / Getting started
+Most of the apps are built with [Express.js](https://expressjs.com/), and the front-end uses the [Nunjucks templating language](https://mozilla.github.io/nunjucks/templating.html) and the [GOV.UK Design System](https://design-system.service.gov.uk/).
 
-In order to get started you will need to run the LTS version of [Node.js](https://nodejs.org/en/).
+- [Getting Started](#getting-started)
+- [Tests](#tests)
+- [API Documentation](#api-documentation)
+- [Style Guide](#style-guide)
 
-Additionaly you can run the entire solution using Docker containers so a local [Docker](https://www.docker.com/products/docker-desktop) instance is required.
+## Getting Started
 
-The repository is structured like a monorepo with two main folders `apps` and `packages`.
+### Prerequisites
 
-`apps` - Holds the main Express.js applications responsible for the DB access API backend and the front facing web app.
-`packages` - Holds the common packages that can be used by all apps.
+* [Node.js](https://nodejs.org/en/) LTS & npm
+* [Docker](https://www.docker.com/products/docker-desktop) (to run a local db)
 
-## Developing
+### Database Setup
 
-### Built With
+#### Database Server Setup
 
-The entire solution is built on top [Express.js](https://expressjs.com/) and [Nunjucks templating language](https://mozilla.github.io/nunjucks/templating.html), both for the web and api backends.
+Use Docker to run an instance of SQL Server, replace "<YourStrong@Passw0rd>":
```

Comment:
Need to ensure the password complexity for SQL server is adequate, on some versions may lead to a misleading "login failed" error.

Reply:
Good point, note added: https://github.com/Planning-Inspectorate/back-office/pull/874/commits/539bc62797a589e78375baa4f1e3e0ea7e6a13b9